### DISCUSSION TITLE
Dependency-Track Working

### DISCUSSION THREAD

**Discussion by bhuvi11** [DependencyTrack/dependency-track#1240] [Q&A] [answered]
Hello Team, 

I am new to SBOM and Dependency Track.
I started Dependency Track using docker-compose and i am able to see the UI.
The only way i can see vulnerabilities is through importing SBOM ? or can i import the jar for the application?

How different is this from Dependency Check? It also shows the vulnerabilities in the dependencies included in the project

**Comment by stevespringett** [comment, marked as answer]
> The only way i can see vulnerabilities is through importing SBOM?

Correct.  Generating build-time SBOMs is a good engineering practice, and requirements of NIST SSDF, OWASP SCVS, and BSIMM

Dependency-Track is a [Component Analysis](https://owasp.org/www-community/Component_Analysis) platform. Dependency-Check is an SCA tool.

See: https://docs.dependencytrack.org/odt-odc-comparison/
